wire-ios-sync-engine/Tests/Source/UserSession/OperationStatusTests.swift

```diff
@@ -52,15 +52,6 @@ class OperationStatusTests: MessagingTest {
         XCTAssertEqual(sut.operationState, .background)
     }
 
-    func testOperationState_whenInBackgroundWithOngoingCall() {
-        // when
-        sut.isInBackground = true
-        sut.hasOngoingCall = true
-
-        // then
-        XCTAssertEqual(sut.operationState, .backgroundCall)
-    }
-
```

Comment:
@David-Henner I think this wasn't supposed to be deleted

Reply:
correct